components/collector/src/source_collectors/azure_devops/issues.py

```diff
@@ -46,8 +46,7 @@ def _item_select_fields(self) -> list[str]:
     async def _get_work_item_responses(self, auth: aiohttp.BasicAuth) -> SourceResponses:
         """Separately get each work item from the API."""
         api_url = await self._api_wit_url(endpoint="workitemsbatch")
-        batch_size = min(self.MAX_IDS_PER_WORK_ITEMS_API_CALL, SourceMeasurement.MAX_ENTITIES)
```

Comment:
Do we not want to retain this dynamic resolve?

Reply:
No, because I think this was a (harmless) bug: why base the batch size with which to get the work items from Azure DevOps Server on the maximum number of measurement entities if we still get all the work items anyway?